pallets/msa/src/weights.rs

```diff
@@ -130,48 +130,52 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Msa DelegatorAndProviderToDelegation (r:1 w:1)
 	// Storage: Schemas CurrentSchemaIdentifierMaximum (r:1 w:0)
 	fn grant_delegation(s: u32, ) -> Weight {
-		Weight::from_ref_time(93_763_000 as u64)
-			// Standard Error: 10_000
-			.saturating_add(Weight::from_ref_time(86_000 as u64).saturating_mul(s as u64))
+		Weight::from_ref_time(94_269_000 as u64)
+			// Standard Error: 12_000
+			.saturating_add(Weight::from_ref_time(101_000 as u64).saturating_mul(s as u64))
 			.saturating_add(T::DbWeight::get().reads(6 as u64))
 			.saturating_add(T::DbWeight::get().writes(2 as u64))
 	}
 	// Storage: Msa PublicKeyToMsaId (r:1 w:0)
 	// Storage: Msa DelegatorAndProviderToDelegation (r:1 w:1)
 	fn revoke_delegation_by_delegator() -> Weight {
-		Weight::from_ref_time(24_100_000 as u64)
+		Weight::from_ref_time(24_643_000 as u64)
 			.saturating_add(T::DbWeight::get().reads(2 as u64))
 			.saturating_add(T::DbWeight::get().writes(1 as u64))
 	}
 	// Storage: Msa PublicKeyToMsaId (r:1 w:0)
 	// Storage: Msa ProviderToRegistryEntry (r:1 w:1)
-	fn create_provider(_s: u32, ) -> Weight {
-		Weight::from_ref_time(21_502_000 as u64)
+	fn create_provider(s: u32, ) -> Weight {
+		Weight::from_ref_time(21_273_000 as u64)
+			// Standard Error: 6_000
+			.saturating_add(Weight::from_ref_time(30_000 as u64).saturating_mul(s as u64))
 			.saturating_add(T::DbWeight::get().reads(2 as u64))
 			.saturating_add(T::DbWeight::get().writes(1 as u64))
 	}
+	// Storage: Msa PayloadSignatureRegistry (r:0 w:1)
 	fn on_initialize(m: u32, ) -> Weight {
```

Comment:
marking.

Reply:
Tracking in issue https://github.com/LibertyDSNP/frequency/issues/785